test/builtin_dataset_mocks.py

```diff
@@ -62,27 +62,47 @@ def _parse_mock_info(self, mock_info):
 
         return mock_info
 
-    def prepare(self, config):
+    def load(self, config):
```

Comment:
this LGTM, but I think it's worth documenting that we need the "temp folder logic" so that the files appear as missing such that the download **+ pre-process** logic gets triggered. Perhaps a permalink to the 3 new lines could be useful